doc/content/devices/models/milesight-em300-th/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Milesight EM300-TH"
+description: ""
+weight: 
+---
+
+{{< figure src="milesight-EM300-TH.png" alt="Milesight EM300-TH" class="float plain" width="75%" >}}
+
+The [Milesight EM300-TH](https://www.milesight.com/iot/product/lorawan-sensor/em300-th) is an environment sensor suitable for monitoring indoor and outdoor temperature and humidity. It can be widely used in office areas, greenhouses and other environments.
+
+<!--more-->
+
+## Provisioning Information
+The provisioning information, **DevEUI**, can be found on the back of the device. You can use `24E124C0002A0001` as the **JoinEUI**. The **AppKey** can be auto-generated within {{% tts %}}.
+
+
+{{< figure src="milesight-EM300-TH-back.png" alt="EM300-TH provisioning information" >}}
+
+
+{{< note "The Milesight EM300-TH doesn’t have a QR code that supports {{% tts %}} to extract provisioning information." />}}
```

Comment:
```suggestion
{{< note "The Milesight EM300-TH cannot be onboarded to {{% tts %}} using QR code." />}}
```

Reply:
Does this device have a QR code? But that QR code cannot be used for onboarding to TTS? I'm asking because if there's no QR code at all, you can remove this part completely, but if there is (but cannot be used to onboard to TT), then you can simplify this as I suggested

Reply:
It has a QR code but doesn't support onboarding.